bluepyentity/app/explorer.py

```diff
@@ -22,6 +23,21 @@ def _tree():
     return defaultdict(_tree)
 
 
+class VimTree(Tree):
+    """Add j/k bindings for tree navigation"""
+
+    BINDINGS = [
+        Binding("enter", "select_cursor", "Select", show=False),
+        Binding("up", "cursor_up", "Cursor Up", show=False),
+        Binding("down", "cursor_down", "Cursor Down", show=False),
+        Binding("k", "cursor_up", "Cursor Up", show=False),
+        Binding("j", "cursor_down", "Cursor Down", show=False),
+        Binding("h", "select_cursor", "Cursor Up", show=False),
+        Binding("l", "select_cursor", "Cursor Down", show=False),
+        Binding("G", "action_scroll_end", "Goto end", show=False),
```

Comment:
I did not get the "G" to work for me. I guess it is a textual issue or something with my terminal

Reply:
yup, don't need the `action_` prefix in bindings, oops.